### PR TITLE
[CR-1637] Add subscription demo to checkout-demo

### PR DESCRIPTION
**Pull request by antonioliv**
Checkout-demo is ready to demonstrate how the subscription works. At a first instance it may not show some pages with information about the frequency and the number of debits to finalize the payment. It will be visible after the update of the Checkout-sdk.

### CODE REVIEW

**Comment by antonioliv** [comment]
Captura de ecrã 2022-08-05, às 17 27 56

**Comment by antonioliv** [comment]
https://www.loom.com/share/cb02ae4decc64cc2b9401f28f08e30fb

**Review by Arlanthir — COMMENTED** [review]
Seems good, but can we take the opportunity to make the strings consistent with `'single-quotes'`?

File: app.js [hunk 1]
```diff
@@ -40,18 +40,30 @@ function createCheckoutSession(req, res) {
...
-      methods: ['cc', 'mb', 'mbw', 'dd', 'vi', 'uf', 'sc'],
-      type: 'sale',
-      capture: {
-        transaction_key: 'string',
-        descriptive: 'Descriptive Example',
-      },
-      currency: 'EUR',
-      expiration_time: tomorrow.toISOString().slice(0, 16).replace('T', ' '), // e.g. 2022-01-02 18:25
+
+  console.log("payment type: ", req.params.type)
```
[the remark this answers is not in the document]
Comment (antonioliv) [reply app.js:44]: Nop, I left it there on purpose. Logs the requested payment type right before the manifest.

**Review by Arlanthir — APPROVED** [review]
LGTM

**Pull request merged**